gr-utils/python/modtool/modtool_rename.py

```diff
@@ -38,6 +41,17 @@ def __init__(self):
         self._skip_cmakefiles = False
         self._license_file = None
 
+    def get_block_candidates(self):
+        cpp_blocks = filter(lambda x: not (x.startswith('qa_') or
+                            x.startswith('test_')),
+                            glob.glob1("lib", "*.cc"))
```

Comment:
Maybe add `*.cpp` to the glob. It's rarely used, and not created by modtool, but it's the more common ending for C++ files and someone might be using it.
Or not. I don't feel strongly about this.

Reply:
I have updated the same for the inclusion of `cpp` files.